SparkPost/matchbox
# UX-720 Fix missing `sizing-550`, Add types to design-tokens

### What Changed
- Adds typescript types to `design-tokens`
- Fixes `sizing-550`, it was missing

### How To Test or Verify
- Pull down this repo, and run `lerna link` (Required)
- Open up a file or component that uses design tokens
- Ensure they are properly typed 

### PR Checklist
